ssts/segmentation/example.py

```diff
@@ -131,6 +138,27 @@ def main():
         # Plot classification distributions
         mm.show_classification_distributions(labels, data)
 
+    ## NOTE Segmentation example removing height property after outlier removal
+    elif example_number == "6":
+
+        # Initialize GMM segmentation
+        pre_seg = seg_gmm.SegmenterGMM(n_components=2, embedding_dim=3)
+
+        # Get outliers
+        outliers = mm.extract_outliers(data)
+
+        # Remove height property
+        no_height_data = np.delete(data, 4, axis=2)
```

Comment:
We can remove the height data from data using as shown above. Then we can use this to run the gmm segmentation class. This works well with the nano wires.